DEPS

```diff
@@ -913,6 +913,16 @@ deps = {
     ],
     'condition': 'download_impeller_cmake_example and host_os == "mac"',
     'dep_type': 'cipd',
+  },
+
+  'src/third_party/flutter_font_fallbacks': {
```

Comment:
we might want to call these something that more clearly indicates their provenance (Google Fonts) and use (testing only), e.g. `google_fonts_for_unit_tests` or something